x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality/data_quality_panel/data_quality_summary/ilm_phase_filter/styles.tsx

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFormControlLayout } from '@elastic/eui';
+import styled from 'styled-components';
+
+export const StyledOption = styled.div`
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  width: 100%;
+`;
+
+export const StyledOptionLabel = styled.span`
+  font-weight: bold;
+`;
+
+export const StyledFormControlLayout = styled(EuiFormControlLayout)`
+  height: 42px;
```

Comment:
Could we use the theme value for this styles? something like `theme.eui.euiFontWeightBold` and `theme.eui.euiSize`

Reply:
fixed, thanks